### ISSUE TITLE
Selected ✅ icon for RadioCardGroup component lacks sufficient icon contrast of 3:1 (in dark mode)

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#778]
expected color for icon in dark mode: `var(--accent-success)`
